livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -198,7 +202,7 @@ def __init__(
         entra_token: str | None = None,
         api_key: str | None = None,
         api_version: str | None = None,
-        base_url: str | None = None,
+        base_url: NotGivenOr[str] = NOT_GIVEN,
```

Comment:
nit, I generally prefer to use `str | None` over NotGivenOr when we don't need to. 
(we tend to over-use NotGiven in our codebase, but it's OK)